### ISSUE TITLE
Pull request angular-ui-calendar so they will publish directly to Meteor

### ISSUE THREAD

**Issue by Urigo** [planet-training/meteor-angular-ui-calendar#1]
Same as here:
https://github.com/planet-training/meteor-angular-material/issues/1

:)

**Comment by dohomi** [comment]
Thanks @Urigo I'll look into it soon. Makes totally sense for sure.

**Comment by dohomi** [comment]
Opened now: https://github.com/angular-ui/ui-calendar/issues/256

But still its not that easy, because of the dependency to some jquery-ui packages and the FullCalendar source itself. Maybe we should need to provide seperate packages for this.

**Comment by Urigo** [comment]
Yes I think you should create packages for them as well and depend on them
